invenio_deposit/ext.py

```diff
@@ -38,6 +40,8 @@ def __init__(self, app=None):
         """Extension initialization."""
         if app:
             self.init_app(app)
+        post_action.connect(index_deposit_after_publish, sender=app,
```

Comment:
No. It should indeed have ``sender``, both both extensions should register the signal on their app. The config variable is in order to allow people not to have the ``index_deposit_after_publish`` called if they want to provide alternative way of indexing the record.